### PR TITLE
Register Subdomain: tusharhero

### PR DESCRIPTION
**Pull request by tusharhero**
Added the domain: tusharhero

### CODE REVIEW

**Review by omsenjalia — DISMISSED** [review]
wait for a maintiner to merge this

**Review by MaskDuck — CHANGES_REQUESTED** [review]
can you also elaborate on what will this domain be used for?

File: domains/tusharhero.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "tusharhero",
+        "email": "tusharmaharana44676@gmail.com"
+      },
+
+      "record": {
+        "TXT": "c84447185947a39adfb909ef58b9e4"
```
[the remark this answers is not in the document]
Comment (tusharhero) [reply domains/tusharhero.json:9]: It will be used for github pages

[the remark this answers is not in the document]
Comment (tusharhero) [reply domains/tusharhero.json:9]: then how do i use it for github pages?

[the remark this answers is not in the document]
Comment (tusharhero) [reply domains/tusharhero.json:9]: https://www.is-a.dev/github_pages

[the remark this answers is not in the document]
Comment (tusharhero) [reply domains/tusharhero.json:9]: Hello?

**Comment by tusharhero** [comment]
Please resolve

**Comment by tusharhero** [comment]
When u approve it

**Comment by tusharhero** [comment]
I have been waiting for like a month

**Comment by mtgsquad** [comment]
Hi, I'm sorry but at the moment, we're unable to merge any PRs, due to issues from our provider "Namecheap". For further information please join our discord server here: [https://s.molai.dev/is-a-dev](https://s.molai.dev/is-a-dev) (Shortened Redirect)

**Review by wdhdev — APPROVED** [review]

**Comment by tusharhero** [comment]
I  don't want to index my site but It is a personal site. What information do I need to add to my site?

**Comment by tusharhero** [comment]
Ok I will do that

**Comment by TheBotlyNoob** [comment]
I'm closing this because of the large amounts of pull requests. You can make another PR later with fixes if you want.

**Pull request closed without merging**